src/server/helpers/utils.js

```diff
@@ -50,7 +50,8 @@ export function getEntityModels(orm: Object): Object {
 		Work
 	};
 }
-function getRevisionModels(orm) {
+
+export function getRevisionModels(orm) {
```

Comment:
Shouldn't this function also jump over to src/server/helpers/revisions.js ?

Reply:
I didn't write this function so I just assumed it's being used for other things too. I guess I am wrong, I will have a look and change if that isn't the case.